applications/FluidDynamicsApplication/custom_elements/vectorial_convection_fractional_element.cpp

```diff
@@ -0,0 +1,725 @@
+        const double clhs0 = N[0]*vconv(0,0) + N[1]*vconv(1,0) + N[2]*vconv(2,0);
+const double clhs1 = N[0]*vconv(0,1) + N[1]*vconv(1,1) + N[2]*vconv(2,1);
+const double clhs2 = DN(0,0)*clhs0 + DN(0,1)*clhs1;
+const double clhs3 = N[0]*bdf0;
+const double clhs4 = clhs2 + clhs3;
+const double clhs5 = 1.0*1.0/(dyn_tau*1.0/dt + stab_c2*1.0/h*sqrt(clhs0*clhs0 + clhs1*clhs1));
+const double clhs6 = clhs5*(DN(0,0)*vconv(0,0) + DN(0,1)*vconv(0,1) + DN(1,0)*vconv(1,0) + DN(1,1)*vconv(1,1) + DN(2,0)*vconv(2,0) + DN(2,1)*vconv(2,1));
+const double clhs7 = N[0]*clhs6;
+const double clhs8 = clhs2*clhs5;
+const double clhs9 = N[0]*clhs2 + bdf0*(N[0]*N[0]) + clhs4*clhs7 + clhs4*clhs8;
+const double clhs10 = N[1]*clhs3;
+const double clhs11 = DN(1,0)*clhs0 + DN(1,1)*clhs1;
+const double clhs12 = N[1]*bdf0;
+const double clhs13 = clhs11 + clhs12;
+const double clhs14 = N[0]*clhs11 + clhs10 + clhs13*clhs7 + clhs13*clhs8;
+const double clhs15 = N[2]*clhs3;
+const double clhs16 = DN(2,0)*clhs0 + DN(2,1)*clhs1;
+const double clhs17 = N[2]*bdf0 + clhs16;
+const double clhs18 = N[0]*clhs16 + clhs15 + clhs17*clhs7 + clhs17*clhs8;
+const double clhs19 = N[1]*clhs6;
+const double clhs20 = clhs11*clhs5;
+const double clhs21 = N[1]*clhs2 + clhs10 + clhs19*clhs4 + clhs20*clhs4;
+const double clhs22 = N[1]*clhs11 + bdf0*(N[1]*N[1]) + clhs13*clhs19 + clhs13*clhs20;
+const double clhs23 = N[2]*clhs12;
+const double clhs24 = N[1]*clhs16 + clhs17*clhs19 + clhs17*clhs20 + clhs23;
+const double clhs25 = N[2]*clhs6;
+const double clhs26 = clhs16*clhs5;
+const double clhs27 = N[2]*clhs2 + clhs15 + clhs25*clhs4 + clhs26*clhs4;
+const double clhs28 = N[2]*clhs11 + clhs13*clhs25 + clhs13*clhs26 + clhs23;
+const double clhs29 = N[2]*clhs16 + bdf0*(N[2]*N[2]) + clhs17*clhs25 + clhs17*clhs26;
+lhs(0,0)=clhs9;
+lhs(0,1)=0;
+lhs(0,2)=clhs14;
+lhs(0,3)=0;
+lhs(0,4)=clhs18;
+lhs(0,5)=0;
+lhs(1,0)=0;
+lhs(1,1)=clhs9;
+lhs(1,2)=0;
+lhs(1,3)=clhs14;
+lhs(1,4)=0;
+lhs(1,5)=clhs18;
+lhs(2,0)=clhs21;
+lhs(2,1)=0;
+lhs(2,2)=clhs22;
+lhs(2,3)=0;
+lhs(2,4)=clhs24;
+lhs(2,5)=0;
+lhs(3,0)=0;
+lhs(3,1)=clhs21;
+lhs(3,2)=0;
+lhs(3,3)=clhs22;
+lhs(3,4)=0;
+lhs(3,5)=clhs24;
+lhs(4,0)=clhs27;
+lhs(4,1)=0;
+lhs(4,2)=clhs28;
+lhs(4,3)=0;
+lhs(4,4)=clhs29;
+lhs(4,5)=0;
+lhs(5,0)=0;
+lhs(5,1)=clhs27;
+lhs(5,2)=0;
+lhs(5,3)=clhs28;
+lhs(5,4)=0;
+lhs(5,5)=clhs29;
+
+        //  Add intermediate results to local system.
+        noalias(rLHS) += lhs * rData.Weight;
+    }
+
+    /***********************************************************************************/
+    /***********************************************************************************/
+    template <>
+    void VectorialConvectionFractionalElement<VectorialConvectionFractionalElementData<3, 4>>::ComputeGaussPointLHSContribution(VectorialConvectionFractionalElementData<3, 4> &rData,
+                                                                                                                                MatrixType &rLHS)
```

Comment:
```suggestion
    void VectorialConvectionFractionalElement<VectorialConvectionFractionalElementData<3, 4>>::ComputeGaussPointLHSContribution(
    VectorialConvectionFractionalElementData<3, 4> &rData,
    MatrixType &rLHS)
```
Please follow the style convention.